packages/fxa-settings/src/components/Settings/PageRecentActivity/index.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RouteComponentProps } from '@reach/router';
+
+import FlowContainer from '../FlowContainer';
+import { useLocalization } from '@fluent/react';
+import { useAccount } from '../../../models';
+import { SecurityEvent as SecurityEventSection } from './SecurityEvent';
+import React, { useState, useEffect } from 'react';
+
+export const PageRecentActivity = (_: RouteComponentProps) => {
+  const account = useAccount();
+  const [securityEvents, setSecurityEvents] = useState(account.securityEvents);
+
+  const { l10n } = useLocalization();
+
+  useEffect(() => {
+    (async () => {
+      const securityEvents = await account.getSecurityEvents();
+      setSecurityEvents(securityEvents);
+    })();
+  });
+
+  return (
+    <FlowContainer
+      title={l10n.getString(
+        'recent-activity-title',
+        null,
+        'Recent Account Activity'
+      )}
```

Comment:
Mind using `ftlMsgResolver` here? We could also go ahead and add the l10n tests in with `testL10n` or `testAllL10n` but I know we'll have tickets later to make sure we're using that everywhere once we fix a couple bugs in those tests.